client/src/Videos.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+function Video(props) {
+    const videoUrl = props.videoObj.url.replace("watch?v=", "embed/");
+
+    return (
+      <div className="videos">
+        {/* delete button */}
```

Comment:
Just a tip - in general I would recommend avoiding leaving comments in your code. Your code should describe itself based on well named components, functions and classes. As your code changes, your comments may become out of date (nothing breaks if you leave an old, invalid comment in there), so commenting can become unreliable. If you feel like you need to leave a comment because you can't remember what a piece of code is doing, chances are it could be extracted into a named component or function to be more descriptive.

Reply:
In this case though, I don't think you need to change the code, it speaks for itself